base_control/base_control.ino

```diff
@@ -0,0 +1,87 @@
+//General
+#include<math.h>
+#define PI_C 3.14159265358979323846
+unsigned long time=0;
+
+//PID
+#include "PID.h"
+
+
+//BNO
+#include <Wire.h>
+#include <Adafruit_Sensor.h>
+#include <Adafruit_BNO055.h>
+#include <utility/imumaths.h>
+#define BNO055_SAMPLERATE_DELAY_MS (100)
+
+Adafruit_BNO055 bno = Adafruit_BNO055(55);
+int BNOSetPoint = 0;
+
+
+
+//Motor
+#define TIME_VELOCITY_SAMPLE 100.0 //millis
+#define PULSES_PER_REVOLUTION 4320.0
+#define WHEEL_DIAMETER 0.1 //meters
+#define MINPWM 100
+#define MAXTICKS 286
+#define MINTICKS 190
+
+#include "Motor.h"
+
+//Movement
+#include "Movement.h"
+Movement moveAll;
```

Comment:
So here we have moveAll. If many classes are gonna use it, better send it as a parameter. It's bad practice to have global variables shared across files.

Reply:
Or any other solution is also acceptable